HISTORY.rst

```diff
@@ -6,13 +6,14 @@ Latest
 ------
 
 * Fix docs to reflect API changes after refactor.
-* Fix wrong initialisation of optical depth for GrayLongwaveRadiation
+* Fix wrong initialisation to use sigma values instead of pressure values 
```

Comment:
initialization, if we're using American spellings.

Reply:
I will retain British spellings, it is hard for me to write American versions without consciously
remembering to do so.

Reply:
OK for documentation, though I will insist on American spellings for function/variable names where it's more likely to cause confusion for users.

Reply:
Yes, I'm fine with that!